ui/components/app/snaps/snap-ui-selector/snap-ui-selector.tsx

```diff
@@ -34,7 +34,7 @@ import { useSnapInterfaceContext } from '../../../../contexts/snaps';
 export type SnapUISelectorProps = {
   name: string;
   title: string;
-  options: string[];
+  options: [{ value: string; disabled: boolean }];
```

Comment:
Is this intended to be a tuple? Seems like it's used as array below.

Reply:
Intended to be an array AFAIK

Reply:
```suggestion
  options: { value: string; disabled: boolean }[];
```